tests/acceptance/features/cliDataExporter/import.feature

```diff
@@ -87,3 +87,10 @@ Feature: An administrator wants to import a user using the commandline
     When a user is imported from path "simpleExport/unknown" using the occ command
     Then the command should have failed with exit code 1
     And the command output should contain the text "user.json not found"
+
+  @issue-210
+  Scenario: importing with empty trash bin
+    When a user is imported from path "emptyTrashbinExport/usertrash" using the occ command
```

Comment:
According to the bug report here https://github.com/owncloud/data_exporter/issues/210 , it should be like this? 
Also let's add the bug demonstration in separate feature file
```suggestion
    When a user is imported from path "emptyTrashbinExport/usertrash" using the occ command
    Then the command should have failed with exit code 1
    And the command output should contain the text "Import failed on ..."
```

Reply:
ok